SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -79,26 +79,26 @@ ServerStatusWatcher {
 	prHandleClientLoginInfoFromServer { |newClientID, newMaxLogins|
 		if (newMaxLogins.notNil) {
 			if (newMaxLogins != server.options.maxLogins) {
-				"%: scsynth has maxLogins % - adjusting my options accordingly.\n"
+				"%: server process has maxLogins % - adjusting my options accordingly.\n"
 				.postf(server, newMaxLogins);
 				server.options.maxLogins = newMaxLogins;
 			} {
-				"%: scsynth maxLogins % match with my options.\n".postf(server, newMaxLogins);
+				"%: server process's maxLogins (%) matches my options.\n".postf(server, newMaxLogins);
```

Comment:
maybe `"%: maxLogins (%) of server process matches my options\n"

Reply:
why? there is no semantic difference

Reply:
yes, I hoped so -- just thought of a way to get around the "process's" s-collisions :)

Reply:
ok, just don't want to get into hair-splitting on something that was meant as a quick fix.

Reply:
sure, not an issue at all!